src/question_baseselect.ts

```diff
@@ -92,6 +95,12 @@ export class QuestionSelectBase extends Question {
     }
     this.removeFromDependedQuestion(this.getQuestionWithChoices());
   }
+  protected getItemValueType() {
+    return "itemvalue";
+  }
```

Comment:
We have the item value type. Probably we could create it via serializer in base item value class...